best_practices/running_workflows.md

```diff
@@ -0,0 +1,126 @@
+# Running workflows
+
+*"I ran your workflow, but it does not work"*
+
+\-- Anonymous
+
+## Record-keeping
+### Keep records of the workflows, inputs and parameters.
```

Comment:
All of that is recorded in Galaxy, does this maybe date from many years back when it wasn't in the user interface ?